api/routers/areas.py

```diff
@@ -55,6 +62,11 @@ async def create_area(new_area: AreaConfigDTO, reboot_processor: Optional[bool]
             status_code=status.HTTP_400_BAD_REQUEST,
             detail=bad_request_serializer("Area already exists", error_type="config duplicated area")
         )
+    elif new_area.id.upper() == "ALL":
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail=bad_request_serializer("Area with ID: 'ALL' is not valid.", error_type="Invalid ID")
```

Comment:
Here and in other places we would use string interpolation with the constant Pablo mentioned